src/main/java/io/github/externschool/planner/service/TeacherServiceImpl.java

```diff
@@ -21,18 +24,17 @@
 @Service
 @Transactional
 public class TeacherServiceImpl implements TeacherService {
-    private final TeacherRepository teacherRepository;
-    private final VerificationKeyRepository keyRepository;
-    private final CourseRepository courseRepository;
```

Comment:
I guess we still need this repo since it's been used in public void deleteTeacherById(Long id)